### DISCUSSION TITLE
Welcome to elFinder Discussions!

### DISCUSSION THREAD

**Discussion by nao-pon** [Studio-42/elFinder#3303] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JupiterJones** [comment]
Hi everyone,

I have been using elFinder for a number of years and support a [Smalltalk backend](https://github.com/JQueryWidgetKit/elFinder) for [GemStone Smalltalk](http://gemtalksystems.com/) running [Seaside](https://github.com/SeasideSt/Seaside)

Great library. Thanks @nao-pon
